prometheus_client/multiprocess.py

```diff
@@ -28,8 +28,11 @@ def collect(self):
             d = core._MmapedDict(f)
             for key, value in d.read_all_values():
                 metric_name, name, labelnames, labelvalues = json.loads(key)
-                metrics.setdefault(metric_name, core.Metric(metric_name, 'Multiprocess metric', typ))
-                metric = metrics[metric_name]
+
+                metric = metrics.get(metric_name)
+                if metric is None:
+                    metrics[metric_name] = metric = core.Metric(metric_name, 'Multiprocess metric', typ)
```

Comment:
Chaining = can be confusing, can you make this two statements?

Reply:
no problem